SPECS/nginx/nginx.spec

```diff
@@ -39,16 +41,35 @@ The nginx-filesystem package contains the basic directory layout
 for the Nginx server including the correct permissions for the
 directories.
 
+%package otel_ngx_module
+License:        Apache-2.0
```

Comment:
Curious, this is a different license from the main package. How does that work? Does this break any of our license checking tools?